tests/components/side-panel/side-panel.spec.tsx

```diff
@@ -12,11 +13,14 @@ describe("SidePanel", () => {
             unobserve: jest.fn(),
             disconnect: jest.fn(),
         }));
+        jest.useFakeTimers();
```

Comment:
by mocking the timers, `waitFor` will actually fail to work properly, hence we should:
- remove mocking the timers
- `mockOnSubmit` will still continue to be mocked with setTimeout, just without a delay so it will be resolved in the next hoop